.env

```diff
@@ -1,3 +1,3 @@
 NEXT_PUBLIC_STAGE=localhost
 NEXT_PUBLIC_MOCK_API_URL=http://localhost:3001
-NEXT_PUBLIC_STAGING_API_URL=http://morris.edpn.io:8080
+NEXT_PUBLIC_STAGING_API_URL=http://duval.edpn.io:8080
```

Comment:
You should probably just dump stuff like this in it's own PR, it'd get waved through and save you having to fix conflicts in all the PRs you add it to